Ginger/Ginger/Run/RunSetsExecutionsHistoryPage.xaml.cs

```diff
@@ -728,29 +782,53 @@ private bool TryAddActivitiesToSharedRepository(IEnumerable<(BusinessFlow,IEnume
 
         private void ExportUseCaseFromBusinessFlowExecutionSequence(BusinessFlowExecutionSequence bfSequence)
```

Comment:
Please move BPMN related methods to helper class

Reply:
Moved this logic to the new `RunSetExecutionHistoryToBPMNExporter` class.